lib/hasher/include/checksum.hpp

```diff
@@ -15,6 +15,9 @@ extern "C" {
 #endif
 
 #define SHA256_CHKSUM_PREFIX "sha2:"
+#define SHA512_CHKSUM_PREFIX "sha-512:"
+#define ADLER32_CHKSUM_PREFIX "adler-32:"
```

Comment:
remove the hyphens?   is there a reason we need them?

Reply:
Not necessarily.  The hyphen is the normal terminology for them and I used it to distinguish it from sha2 which is also known as sha-256 when it references a specific algorithm rather than the family of sha-2 algorithms.

However, it is known as "sha-2" and we use "sha2" so it makes sense to keep this consistent.  I will remove the hashes.

Reply:
Done